### DISCUSSION TITLE
Allow output port when no schedules are defined in the scheduler node

### DISCUSSION THREAD

**Discussion by zachowj** [jensrossbach/node-red-contrib-chronos#109] [Ideas & Requests]
Would like to see an option to always have an output even if no events are defined for the scheduler node.

One of my use cases is to dynamically generate a list of events from an external source and pass it into the scheduler node. When there are no predefined events the node has no output.

**Comment by jensrossbach** [comment]
Hi, I need to think about it. The original idea of the dynamic inputs was to override existing events defined in the editor but not to define new events in the runtime. Having a single output for completely dynamic events might be feasible but such feature would probably not be compatible with the option to have one output port per event (because output ports can only be defined in the editor but not in the runtime).

**Comment by zachowj** [comment]
It might be more appropriate for me to use a different node built for my use case.

**Reply by jensrossbach** [reply]
If there is such node existing, feel free to use. I will continue to think about your proposal but such change would not be trivial as the current logic relies on the fact that the configured schedule is not empty. So careful testing would be required afterwards.
